.github/workflows/doc.yml

```diff
@@ -0,0 +1,39 @@
+on:
+  push:
+    branches:
+      - master
+      - '0.[0-9]+.x'
+      - '1.[0-9]+.x'
+
+
+name: Publish Docs
+
+jobs:
+  publish_docs:
```

Comment:
Would it be possible to guard the publish docs workflow with something like
`if ${GITHUB_REPOSITORY} == "diesel-rs/diesel"`
so that workflows continue to work with forks?

Reply:
I managed to do this in a fork, will have to add a PR here to test?